graphql/mutation_resolver.go

```diff
@@ -305,6 +305,18 @@ func (r *mutationResolver) EnqueuePatch(ctx context.Context, patchID string, com
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error getting patch '%s'", patchID))
 	}
 
+	projectID := utility.FromStringPtr(existingPatch.ProjectId)
+	proj, err := data.FindProjectById(projectID, false, false)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error getting project '%s': %s", projectID, err.Error()))
+	}
+	if proj == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("project '%s' not found", projectID))
+	}
+	if proj.CommitQueue.MergeQueue == model.MergeQueueGitHub {
+		return nil, Forbidden.Send(ctx, "Can't enqueue patches for projects with GitHub merge queue. Click the merge button on the PR instead.")
+	}
+
```

Comment:
added to the mutation resolver as well so that patches with GitHubPRRequester below should not be allowed to be enqueued if github merge queue is enabled